packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -36,6 +36,7 @@ const LARGE_NEGATIVE_DISTANCE_FROM_CENTER = -999999999;
 const _allInstances: {
   [key: string]: FocusZone;
 } = {};
+const _outerZones: Set<FocusZone> = new Set();
```

Comment:
Should this be static?

Reply:
Saves bytes to just use file scope.

Reply:
`_outerZones` will compress to `a`.

`FocusZone.outerZones` will compress to `a.outerZones`

Reply:
Oops sorry, I thought this was in the class scope. Nevermind